### DISCUSSION TITLE
v1.23.1

### DISCUSSION THREAD

**Discussion by adriankumpf** [teslamate-org/teslamate#1663] [General]
### Improvements and Bug Fixes

- Tesla has removed the captcha again …
- Fix error when changing the language to Chinese

#### Translations

- Update Swedish translations ([#1655](https://github.com/adriankumpf/teslamate/pull/1655) by [tobiasehlert](https://github.com/tobiasehlert))

<hr /><em>This discussion was created from the release <a href='https://github.com/adriankumpf/teslamate/releases/tag/v1.23.1'>v1.23.1</a>.</em>

**Comment by cpav1978** [comment]
Hi, I had to reinstall my Docker during hd replacement and have the captcha error again. :-( 
"Error: Captcha stimmt nicht überein"
Do you have any help for me?
If it helps, I have MFA activated in my Tesla account.

**Reply by jun3280net** [reply]
> Hi, I had to reinstall my Docker during hd replacement and have the captcha error again. :-(
> "Error: Captcha stimmt nicht überein"
> Do you have any help for me?
> If it helps, I have MFA activated in my Tesla account.

Yah - there must be something recently changed.  I don't use MFA and still have the problem which I've submitted here : 

#https://github.com/adriankumpf/teslamate/issues/1888
